docs-new/docs/graphql/core/api-reference/schema-api/index.mdx

```diff
@@ -0,0 +1,93 @@
+---
+description: Hasura schema API reference
+keywords:
+  - hasura
+  - docs
+  - schema API
+  - API reference
+slug: index
+---
+
+# Schema API Reference
+
+## Introduction
+
+The schema API provides the following features:
+
+1.  Execute SQL on the underlying Postgres database, supports schema
+    modifying actions.
+
+This is primarily intended to be used as an `admin` API to manage the
+Hasura schema.
+
+:::info Supported from
+
+The schema API is supported for versions `v2.0.0` and above and replaces
+the older [schema/metadata API](/graphql/core/api-reference/schema-metadata-api/index.mdx).
+
+:::
+
+## Endpoint
+
+All requests are `POST` requests to the `/v2/query` endpoint.
+
+## Request structure
+
+```http
+POST /v2/query HTTP/1.1
+
+{
+   "type": "<query-type>",
+   "args": <args-object>
+}
+```
+
+### Request body
+
+[Query](#schema-query)
+
+#### Query{#schema-query}
+
+| Key     | Required | Schema     | Description                     |
+|---------|----------|------------|---------------------------------|
+| type    | true     | String     | Type of the query               |
+| args    | true     | JSON Value | The arguments to the query      |
+| version | false    | Integer    | Version of the API (default: 1) |
+
+## Request types
+
+The various types of queries are listed in the following table:
+
+| `type`                                                                               | `args`                                                                                       | `version` | Synopsis                                      |
+|--------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------|-----------|-----------------------------------------------|
+| **bulk**                                                                             | [Query](#query) array                                                                        | 1         | Execute multiple operations in a single query |
+| [run_sql](/graphql/core/api-reference/schema-api/run-sql.mdx#schema-run-sql}) | [run_sql_args](/graphql/core/api-reference/schema-api/run-sql.mdx#schema-run-sql-syntax)     | 1         | Run SQL directly on Postgres                  |
```

Comment:
stray `}` at end of `run_sql` link - please remove

Reply:
fixed